pydarn/io/dmap.py

```diff
@@ -302,14 +311,14 @@ def test_initial_data_integrity(self):
         zero_check : raises ZeroByteError
         byte_check : raises MistmatchByteError
         """
-        pydarn_logger.debug("Testing the integrity of the dmap file/stream")
+        #pydarn_logger.debug("Testing the integrity of the /stream")
```

Comment:
Probably want to remove this or uncomment and write a more descriptive debug message.

Reply:
Here and elsewhere.

Reply:
Woops for got to uncomment those in. 
Just to avoid repeated information, the debug information follows each other so you would get the file information in earlier messages. This one is indicating the function was called. Then later any error raised or other messages will be presented.